docs/.vuepress/components/NpmUsage.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div>
+    <div v-if="loading">
+      Loading...
+    </div>
+
+    <div v-if="rejected">
+      <p>Unable to get usage statistics for {{ routerName }}</p>
+    </div>
+
+    <div v-if="success">
+      <p>
+        <b>Usage stats:</b> {{ usageCount }} downloads (<a target="_blank" :href="'https://www.npmtrends.com/' + routerName">see more</a>)
+      </p>
+      <p>
+        <b>Github stars:</b> {{ stars }}
+      </p>
+      <p v-if="usageCount < 100">
```

Comment:
Do you any rationale behind 100?

Reply:
No, I just went for a random number, thats why I mentioned in the OP that we should discuss what we should base this treshold on 😛